contribs/gmf/src/services/querymanager.js

```diff
@@ -124,7 +124,7 @@ gmf.QueryManager.prototype.createSources_ = function(node) {
         var childLayerNames = [];
         node.childLayers.forEach(function(childLayer) {
           if (childLayer.queryable) {
-            childLayerNames.push(childLayer.layers);
+            childLayerNames.push(childLayer.name);
```

Comment:
At least that is returned by https://geomapfish-demo.camptocamp.net/2.1/wsgi/themes?background=background